### PR TITLE
Add programari lliure book (#1)

### PR DESCRIPTION
**Pull request by jordimas**
Add programari lliure book

### CODE REVIEW

**Comment by jordimas** [comment]
Add the author of this work and I license it with CC-0 license. @jaumeortola helped to postprocess and review this already

File: server/data/ca/programari-lliure-llibre.txt [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+Gràcies a la disponibilitat del codi font, el programari lliure garanteix una independència respecte al proveïdor.
+Han estat nombrosos els casos d'empreses i usuaris que han estat demandats per intercanvi il·legal de fitxers.
+Hem vist com el programari lliure ja ha canviat la forma d'entendre la indústria de la programació.
+Hi ha qui creu, erròniament, que la venda de llicències és una font extraordinària d'ingressos per a qualsevol empresa.
+Hi ha una correlació directa entre el nombre d'usuaris que té una aplicació lliure i la comunitat que li dóna suport.
+IBM ha demostrat que està al costat del programari lliure en defensar Linux davant els tribunals.
+Integritat del codi font original.
+Internet ha creat els seus propis mecanismes per evitar aquesta mena de situacions.
+L'administració ha de donar un servei universal i, per tant, respectar i impulsar els estàndards oberts en adreçar-se als seus ciutadans.
+L'any mil nou-cents noranta-vui va ser un any especialment important per al programari lliure.
```
Comment (jaumeortola) [inline server/data/ca/programari-lliure-llibre.txt]: @jordimas Cal corregir: noranta-vui > noranta-vuit

Comment (jordimas) [reply server/data/ca/programari-lliure-llibre.txt]: Fet (done)

**Comment by mikehenrty** [comment]
Great! Is this ready for merge @jordimas @jaumeortola ?

**Comment by jordimas** [comment]
Good to go here :+1:

**Comment by mikehenrty** [comment]
Thanks both!

**Pull request merged**